manifests/repo.pp

```diff
@@ -1,12 +1,12 @@
 # PRIVATE CLASS: do not use directly
 class mongodb::repo (
-  $ensure         = $mongodb::params::ensure,
-  $version        = $mongodb::params::version,
+  Variant[Enum['present', 'absent'], Boolean] $ensure = 'present',
+  Optional[String] $version = undef,
   $repo_location  = undef,
   $proxy          = undef,
   $proxy_username = undef,
   $proxy_password = undef,
```

Comment:
can you please align the `=` please?

Reply:
Should we align or remove all alignment? With puppet data types I think alignment makes it harder to read.